### DISCUSSION TITLE
JMA API current weather and forecast data

### DISCUSSION THREAD

**Discussion by Calsy** [open-meteo/open-meteo#802] [Q&A]
Regarding the JMA API.  When requesting the current weather for a particular location in Japan, is the data from the closest JMA observation station provided OR is it an estimation based on the high resolution forecast models?

How are the current weather and forecast models created specifically for Japan?

**Comment by patrick-zippenfenig** [comment]
Hi,
it is simulation based using the high resolution JMA MSM model. In the future, additional realtime/nowcast datasets based in satellite and radar will be implemented.
